### DISCUSSION TITLE
EditorConfig UI: Change settings?

### DISCUSSION THREAD

**Discussion by BhaaLseN** [dotnet/roslyn#53791] [Q&A] [answered]
There are two things I'm wondering about in the new EditorConfig UI:

1. Can I affect how the `.editorconfig` file is saved? I used the Roslyn `.editorconfig` as a base, and generally liked how there are spaces around the equals sign:
    ```
    setting1 = value
    setting2 = value
    ```
    However, updating it with the EditorConfig UI only puts the space in front:
    ```diff
     setting1 = value
     setting2 = value
    +setting3 =value
    ```
    ...and that looks really ugly, especially when it _now_ causes the file to use mixed conventions.
2. Can I change the _language_ being used on the Analyzers tab?
    My Windows is German, but I use English as primary language across pretty much anything else (which includes Visual Studio; where Tools > Options, Environment > International Settings is set to English). Exceptions etc. are in English as well (which usually helps when searching for things), but the "Analyzers" tab insists on showing me (sometimes IMO cringy) German.
    <sup>_(I should probably just go and install a language pack for Windows instead, but it seems odd that this is the only location insisting on using my OS language)_</sup>

**Comment by jmarolf** [comment, marked as answer]
Neither of these are configurable today. Please file this as an issue under https://github.com/dotnet/roslyn/issues/new

**Reply by BhaaLseN** [reply]
Will do, I thought I'd just ask in a discussion to see if I just missed something. Thanks!

**Reply by jmarolf** [reply]
No problem, thanks for filing the issues
